force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -319,16 +414,27 @@ public with sharing class ClassesObjectsMethods {
 
     public static List<String> personDetails() {
         List<String> results = new List<String>();
+
         // Create a new instance of Person class
-        // Name the person 'John Doe' and set the age to 28
+        Person me = new Person('John Doe', 28);
 
+        // Name the person 'John Doe' and set the age to 28
+        me.name = 'John Doe';
+        me.age  = 28;
+                
         // Add the details of the person using the introduceYourself method
+        me.introduceYourself();
+        results.add(me.introduceYourself());
 
         // Get the name of the person using the getter method and add it to the list
+        String myName = me.getName();
+        results.add(me.name);
 
         // Get the age of the person using the getter method and add it to the list
-        
-        return null; // Replace null with the variable you used to store the result
+        Integer myAge = me.getAge();
+        results.add(me.age.toString());
```

Comment:
You have some variables that are declared here but never used. 
Also, try to use the valueOf() method for the age.